authn/keychain_test.go

```diff
@@ -132,7 +132,7 @@ func TestVariousPaths(t *testing.T) {
 		content: `{"credHelpers": {"https://test.io": "test"}}`,
 		check:   checkHelper,
 	}, {
-		content: `{"credStore": "test"}`,
+		content: `{"credsStore": "test"}`,
```

Comment:
This is a good catch and totally my bug (copying [from here](https://github.com/google/containerregistry/blob/master/client/docker_creds_.py#L251-L252)).  thanks for finding and fixing.